docker/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env sh
+set -eu
+
+UID=$(id -ru)
+if [ $UID -eq 0 ]; then
+  adduser -D koishi || true
+  mkdir -p /koishi
+  chown -R koishi /koishi
+  exec su koishi -c "$0 $*"
```

Comment:
why this line? in addition, as puppeteer plugin auto-adds `--no-sandbox`, this part could be removed.